infra/prover/components/bentoBroker.ts

```diff
@@ -457,12 +457,7 @@ export HOME=/home/ubuntu
 chmod +x /local/boundless/scripts/setup.sh
 /local/boundless/scripts/setup.sh
 
-# Install nvcc. After reboot we will set the NVCC flags in the compose.yml using this.
-sudo apt install nvidia-cuda-toolkit -y
```

Comment:
I'm about 4/5 confidence this isn't needed, but can test manually if we want to be absolutely sure it's not